sources/Adapters/picoTracker/gui/picoTrackerEventManager.cpp

```diff
@@ -17,8 +17,15 @@ repeating_timer_t picoTrackerEventManager::timer_ = repeating_timer_t();
 
 uint16_t gTime_ = 0;
 
+picoTrackerEventQueue *queue;
```

Comment:
@democloid I've removed the nasty loop hack and now use the existing timer & event queue for the clock callbacks, hope this is what you had in mind?